presto-orc/src/test/java/com/facebook/presto/orc/TestSelectiveOrcReader.java

```diff
@@ -1007,4 +1016,15 @@ private static String toCharValue(Object value, int minLength)
         }
         return values;
     }
+
+    private static List<Long> varintScaleSequence()
+    {
+        List<Long> values = new ArrayList<>();
+        long[] numbers = new long[] {1L, 1L << 8, 1L << 13, 1L << 20, 1L << 27, 1L << 34, 1L << 40, 1L << 47, 1L << 53, 1L << 60, 1L << 63};
+        for (int i = 0; i < 30000; i++) {
```

Comment:
instead of hard-coding number of rows, add an `int numRows` parameter and call it with `NUM_ROWS`

```
        List<Long> values = varintScaleSequence(NUM_ROWS);
...
    private static List<Long> varintScaleSequence(int numRows)
    {
        List<Long> values = new ArrayList<>();
        long[] numbers = new long[] {1L, 1L << 8, 1L << 13, 1L << 20, 1L << 27, 1L << 34, 1L << 40, 1L << 47, 1L << 53, 1L << 60, 1L << 63};
        for (int i = 0; i < numRows; i++) {
            values.add(numbers[i % numbers.length] + i);
            values.add(-numbers[i % numbers.length] + i);
        }
        return values;
    }
```